packages/material-ui/scripts/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+
+const input = './src/index.js';
+
+const name = 'material-ui';
+
+const globals = {
+  react: 'React',
+  'react-dom': 'ReactDOM',
+};
+
+const getBabelOptions = () => ({
```

Comment:
Something is off. Some babel plugins declared in .babelrc aren't applied. For instance, we still have the `data-mui-test` properties.

Reply:
Yeah, current babelrc is a mess to be honest. It's better to specify targets in browserlist file and completely separate environments with only NODE_ENV.

Reply:
It seems `NODE_ENV=production` isn't taken into account by rollup-babel. But I could be wrong.

Reply:
Its done by rollup plugin replace.

Reply:
That's why sizes of bundles are different